src/scriptworker/constants.py

```diff
@@ -441,13 +441,15 @@
                             # Which repos can perform release actions?
                             # XXX remove /projects/maple when we have a
                             #     different prod signing testing solution
-                            "all-release-branches": ("/releases/mozilla-beta", "/releases/mozilla-release", "/releases/mozilla-esr68", "/projects/maple",),
+                            "all-release-branches": ("/releases/mozilla-beta", "/releases/mozilla-release",
+                                                     "/releases/mozilla-esr68", "/releases/mozilla-esr78", "/projects/maple",),
                             # Limit things like pushapk to just these branches
                             "release": ("/releases/mozilla-release",),
                             "beta": ("/releases/mozilla-beta",),
                             "beta-or-release": ("/releases/mozilla-beta", "/releases/mozilla-release"),
-                            "esr": ("/releases/mozilla-esr68",),
+                            "esr": ("/releases/mozilla-esr68", "/releases/mozilla-esr78",),
                             "esr68": ("/releases/mozilla-esr68",),
+                            "esr78": ("/releases/mozilla-esr78",),
```

Comment:
We might not need `esr78` as a key. We added `esr68` specifically for Fennec scopes.